core/dwc_occurrence_2020-10-24.xml

```diff
@@ -0,0 +1,205 @@
+    <property group='Occurrence' name='reproductiveCondition' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/reproductiveCondition' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#reproductiveCondition' dc:description='The reproductive condition of the biological individual(s) represented in the Occurrence. Recommended best practice is to use a controlled vocabulary.' examples='&quot;non-reproductive&quot;, &quot;pregnant&quot;, &quot;in bloom&quot;, &quot;fruit-bearing&quot;' required='false'/>
+    <property group='Occurrence' name='behavior' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/behavior' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#behavior' dc:description='A description of the behavior shown by the subject at the time the Occurrence was recorded.  Recommended best practice is to use a controlled vocabulary.' examples='&quot;roosting&quot;, &quot;foraging&quot;, &quot;running&quot;' required='false'/>
+    <property group='Occurrence' name='establishmentMeans' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/establishmentMeans' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#establishmentMeans' dc:description='Statement about whether an organism or organisms have been introduced to a given place and time through the direct or indirect activity of modern humans. Recommended best practice is to use controlled value strings from the http://rs.tdwg.org/dwcem/ controlled vocabulary designated for use with this term. For details, refer to https://doi.org/10.3897/biss.3.38084' examples='&quot;native&quot;, &quot;nativeReintroduced&quot;, &quot;introduced&quot;, &quot;introducedAssistedColonisation&quot;, &quot;vagrant&quot;, &quot;uncertain&quot;' required='false'/>
+    <property group='Occurrence' name='degreeOfEstablishment' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/degreeOfEstablishment' dc:relation='http://rs.tdwg.org/dwc/terms/index.htm#degreeOfEstablishment' dc:description='The degree to which an Organism survives, reproduces, and expands its range at the given place and time. Recommended best practice is to use controlled value strings from the http://rs.tdwg.org/dwcdoe/ controlled vocabulary designated for use with this term. For details, refer to https://doi.org/10.3897/biss.3.38084' examples='&quot;native&quot;, &quot;captive&quot;, &quot;cultivated&quot;, &quot;released&quot;, &quot;failing&quot;, &quot;casual&quot;, &quot;reproducing&quot;, &quot;established&quot;, &quot;colonising&quot;, &quot;invasive&quot;, &quot;widespreadInvasive&quot;' required='false'/>
```

Comment:
Same here, I think we link to a thesaurus, either a new xml one hosted here at rs.gbif.org, or elsewhere.